libraries/rush-lib/src/logic/RushConstants.ts

```diff
@@ -132,6 +132,12 @@ export class RushConstants {
    */
   public static readonly repoStateFilename: string = 'repo-state.json';
 
+  /**
+   * The filename ("rush-custom-tips.json") for a file used by Rush to
+   * print user-customized messages
+   */
+  public static readonly customTipsFilename: string = 'rush-custom-tips.json';
```

Comment:
Did this get added to the list of files allowed in common/config/rush?

Reply:
Do you mean I should add the file to common/config/rush?

Reply:
Ah, I missed that you are looking for this file in the root of the repository instead of somewhere in the `common/config` tree. Normally we stash such files in there, but there's a validator that limits which file names are allowed in the `common/config/rush` folder.

Reply:
Fixed